lexers/c/caddyfile.go

```diff
@@ -0,0 +1,202 @@
+package c
```

Comment:
```suggestion
package caddyfile
```

Maybe?

Reply:
No it's in the `/c/` directory, has to be this way.